mvvmfx/src/main/java/de/saxsys/mvvmfx/internal/viewloader/FxmlViewLoader.java

```diff
@@ -79,40 +105,53 @@
     /**
      * This method is used to create a String with the path to the FXML file for
      * a given View class.
-     * 
+     *
+     *
      * This is done by taking the package of the view class (if any) and replace
      * "." with "/". After that the Name of the class and the file ending
      * ".fxml" is appended.
-     * 
+     *
+     * If the View class is annotated with @FxmlPath then the String path supplied
+     * in the annotation  value will be used.
+     *
      * Example: de.saxsys.myapp.ui.MainView as view class will be transformed to
      * "/de/saxsys/myapp/ui/MainView.fxml"
-     * 
+     *
      * Example 2: MainView (located in the default package) will be transformed
      * to "/MainView.fxml"
-     * 
+     *
      * @param viewType
      *            the view class type.
      * @return the path to the fxml file as string.
      */
     private String createFxmlPath(Class<?> viewType) {
+        String emptyPath = "";
+
         final StringBuilder pathBuilder = new StringBuilder();
 
-        pathBuilder.append("/");
+        FxmlPath annotation = viewType.getDeclaredAnnotation(FxmlPath.class); //Get annotation from view
+        String path = annotation != null ? annotation.value() : emptyPath; //Extract path  if present
 
-        if (viewType.getPackage() != null) {
-            pathBuilder.append(viewType.getPackage().getName().replaceAll("\\.", "/"));
+        if (!emptyPath.equalsIgnoreCase(path)) {
```

Comment:
In my oppinion using the "emptyPath" variable makes the code more difficult to understand.
I would use empty string literal `""` in the assignment on line 132 
and check for `path != null && path.trim().isEmpty()` in the conditional on line 135
This takes into account that the value from the annotation could also be `null`.

Another (in my oppinion even more readable) approach is to use `java.util.Optional`:
```java
final FxmlPath pathAnnotation = viewType.getDeclaredAnnotation(FxmlPath.class);
final String fxmlPath = Optional.ofNullable(pathAnnotation)
    .map(annotation -> annotation.value())
    .map(path -> path.trim())
    .orElse("");

if(fxmlPath.isEmpty()) {
...

```